src/platform_impl/macos/window.rs

```diff
@@ -205,6 +205,13 @@ fn create_window(
       ns_window.setAcceptsMouseMovedEvents_(YES);
 
       if pl_attrs.titlebar_transparent {
+        let mut style_mask = ns_window.styleMask();
+        style_mask.set(
+          NSWindowStyleMask::NSFullSizeContentViewWindowMask,
+          true,
+        );
+        ns_window.setStyleMask_(style_mask);
+
```

Comment:
There is `WindowBuilderExtMacOS::with_fullsize_content_view` for exactly doing this. Users should just use it.
```suggestion
```